msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -594,7 +594,8 @@ The following illustrates an example of suggested actions:
 
 > [!NOTE]
 >
-> * `SuggestedActions` are only supported for one-on-one chat bots and text based messages and not for Adaptive Cards or attachments.
+> * `SuggestedActions` are only supported for one-on-one chat bots with both text based messages and Adaptive Cards.
+> * `SuggestedActions` aren't supported for one-on-one chat bots with attachments.
```

Comment:
How about we also add "and other conversation types" after "attachments".
Right now, Suggested action only work in 1:1 with text message and adaptive card, for all other scenarios, it does not support.

Reply:
Updated.